tests/linearizability/linearizability_test.go

```diff
@@ -177,7 +179,8 @@ func TestLinearizability(t *testing.T) {
 				waitBetweenTriggers: waitBetweenFailpointTriggers,
 			}, *scenario.traffic)
 			forcestopCluster(clus)
-			validateWatchResponses(t, watchResponses)
+			watchProgressNotifyEnabled := clus.Cfg.WatchProcessNotifyInterval != 0
```

Comment:
Timed progress notification is always enabled. Users just can set a different interval. So the name `watchProgressNotifyEnabled` is a little misleading.

Reply:
It's not though, he's comparing the interval to a zero value, and if it is a zero value, timed progress notifications are effectively disabled.

Reply:
I was saying etcdserver always enables the timed progress notification. Are you talking about the same thing?

Reply:
Note: etcdserver always triggers the periodical progress notification for each established watch stream, and it sends progress notification for each watch substream if [ProgressNotify](https://github.com/etcd-io/etcd/blob/39d98522d6b7e3ac4463857d9e7db78eb76131b3/api/etcdserverpb/rpc.pb.go#L1995) is `true`.

Reply:
@ahrtr is right, will rephrase it.